tardis/io/util.py

```diff
@@ -90,13 +102,14 @@ def mapping_constructor(self, node):
         return OrderedDict(self.construct_pairs(node))
 
 YAMLLoader.add_constructor(u'!quantity', YAMLLoader.construct_quantity)
+YAMLLoader.add_constructor(u'!demo', YAMLLoader.demo_constructor)
 YAMLLoader.add_implicit_resolver(u'!quantity',
                                  MockRegexPattern(quantity_from_str), None)
 YAMLLoader.add_implicit_resolver(u'tag:yaml.org,2002:float',
-                                 MockRegexPattern(float), None)
+                                                                          MockRegexPattern(float), None)
 YAMLLoader.add_constructor(yaml.resolver.BaseResolver.DEFAULT_MAPPING_TAG,
                            YAMLLoader.mapping_constructor)
-
+YAMLLoader.add_implicit_resolver(u'!demo', re.compile(r'^\d+%\d+%\d+%\d+$'))
```

Comment:
Your `demo` constructor, except from an inappropriate name, doesn't seem to do anything useful.